.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Can you pull out this `eslint` stuff? We don't have a consistent policy on linting, so until we do I want to avoid it altogether.